packages/backend-defaults/src/entrypoints/cache/CacheManager.ts

```diff
@@ -89,22 +95,81 @@ export class CacheManager {
       }
     }
 
+    // Read store-specific options from config
+    const storeOptions = CacheManager.parseStoreOptions(store, config, logger);
+
     return new CacheManager(
       store,
       connectionString,
       options.onError,
       logger,
       defaultTtl,
+      storeOptions,
     );
   }
 
+  /**
+   * Parse store-specific options from configuration.
+   *
+   * @param store - The cache store type ('redis', 'memcache', or 'memory')
```

Comment:
I had originally written some generic wrappers for other cache types - I could make this doc more apparent we're only accepting additional options for redis?

Reply:
Do you have any thoughts here? if we wanted to evaluate inside the `createRedisStoreFactory` we would still need a reference to the root config, or subconfig. 

I don't see a clean way to isolate this without spreading out where we touch the config